### ISSUE TITLE
The mousedown event on body stops working when this addon is used

### ISSUE THREAD

**Issue by titel** [lolmaus/jquery.dragbetter#7]
The plugin seems to be interfering with the triggering of mousedown events on the body but couldn't figure it out how.

Specifically a handler I have set for the body's mousedown event is not triggered when the plugin is included in the page but works fine when not...

Any idea about how should I go about figuring this out? Are there any known side effects of the plugin in regards to how other event are being handled in the page?

Thanks for a great plugin!

**Comment by lolmaus** [comment]
Hey @titel!

Not sure what's going on there. This plugin neither interacts with `body` no listens to the `mousedown` event.

Does the problem happen when you include the plugin but not apply it to any element?

**Comment by titel** [comment]
No, the issue does not occur when the plugin is only loaded but not initialised. I assume is not this plugin specifically but the mix of plugins I have on the page :( I can send a link showing the unwanted behaviour in private if you think that would help.

**Comment by lolmaus** [comment]
@titel Sorry for not responding. Have you been able to figure it out?
